src/version.cpp.in

```diff
@@ -1,6 +1,6 @@
 #define DEF_MONERO_VERSION_TAG "@VERSIONTAG@"
 #define DEF_MONERO_VERSION "0.15.0.0"
-#define DEF_MONERO_RELEASE_NAME "Carbon Chamaeleon"
+#define DEF_MONERO_RELEASE_NAME "Nitrogen Nebula"
```

Comment:
Still at 0.15.0.0

Reply:
Yep, until #6514 is merged

Reply:
OK, I see the same was already merged to the branch now.